dwave/system/samplers/clique.py

```diff
@@ -184,6 +185,18 @@ def __init__(self, *,
         self.child = DWaveSampler(
             failover=failover, retry_interval=retry_interval, **config)
 
+    def close(self):
```

Comment:
Needs to be added to [here](https://github.com/dwavesystems/dwave-system/blob/1bb84c6e73513c66f31dc4bee06aeab6b9e6743d/docs/reference/samplers.rst?plain=1#L87)

Reply:
Do we generally want to keep these methods sorted by significance or alphabet? Because I see we're not consistent in that file.

Reply:
Mostly we sort alphabetically

Reply:
Of 12 lists in that file, only 1 is sorted alphabetically.

Reply:
That's fine, I just added `close()` to end. It makes sense not to be listed as the first method (usually `sample()` is listed first), as long as we're not strictly adhering to alphabetical order.